scripts/eessi-upload-to-staging

```diff
@@ -263,7 +270,7 @@ for file in "$*"; do
         # 1st sign artefact, and upload signature
         if [[ "${sign}" = "1" ]]; then
           # sign artefact
-          ${sign_script} sign ${sign_key} ${file}
+          ${sign_script} sign ${sign_key} ${file} ${bot_instance}
```

Comment:
I think the arguments are now in a somewhat random order, which  is quite brittle. I will open a PR next week for the script to take proper arguments (`--namespace=<namespace>` etc.). That won't affect the format of the signatures though, so doesn't need to affect the ability to move forward with this PR and the general approach.